extension.html

```diff
@@ -52,7 +52,29 @@ <h2>Introduction</h2>
         This document reflects incubation for APIs which are intended to be part of a future version of the Pointer Events specification. They are not yet broadly supported by implementations.
       </p>
       <p>
-	The <code>getCoalescedEvents</code> API (similar to Android <a href="https://developer.android.com/reference/android/view/MotionEvent.html#getHistoricalX(int, int)">getHistorical</a> APIs and iOS <a href="https://developer.xamarin.com/api/member/MonoTouch.UIKit.UIEvent.GetCoalescedTouches/p/MonoTouch.UIKit.UITouch/">GetCoalescedTouches</a> API) exposes all the events that were coalesced into a single event. Coalescing events into a single event enables user agents to prevent processing of the events that don’t get a chance to update the frame content visually. Basically if the javascript handlers process the events and update the screen contents faster than refresh rate only the last update will be rendered and shown in the next frame. However, some applications like drawing apps may want to take advantage of the precise history of events as user moves the pointer on the screen to be able to draw smoother and better looking curves. This API can be used in those scenarios to get all the coalesced events from the event that was dispatched to the javascript.
+	The <code>getCoalescedEvents</code> API (similar to Android <a href="https://developer.android.com/reference/android/view/MotionEvent.html#getHistoricalX(int, int)">getHistorical</a> APIs
+        and iOS <a href="https://developer.xamarin.com/api/member/MonoTouch.UIKit.UIEvent.GetCoalescedTouches/p/MonoTouch.UIKit.UITouch/">GetCoalescedTouches</a> API) exposes all the events
+        that were coalesced into a single event.
+        Coalescing events into a single event enables user agents to prevent processing of the events that don’t get a chance to update the frame content visually.
+        Basically if the javascript handlers process the events and update the screen contents faster than the screen refresh rate only the last update will be rendered and shown in the next frame.
+        So for example instead of sending a few <code>pointermove</code> events in the same frame UA can choose to coalesce all the move events happened in a frame into one event and send only
+        that event in a frame representing the last state of the pointer and javascript will react to that.
+        This would work fine with most of the applications and would save a lot of redundant processing on the web.
```

Comment:
Most applications care only about the last cooridinates and not the intermediate coordinates that occured. However, a class of applications such as drawing applications may wish precise (subframe timing) coordinates in order to draw better detailed lines.